File: C++/Lista_03(Estrutura_Repetitiva)/Soma_Impares.cpp
#include <iostream>

using namespace std;

int main(){

    int x, y, soma=0;

    cout << "Digite dois numeros:" << endl;
    cin >> x;
    cin >> y;

    if (x>y){
        for(int i=y+1;i<x;i++){
            if(i%2!=0){
                soma += i;
            }
        }
    }
    else{
        for(int i=x+1;i<y;i++){
            if(i%2!=0){
                soma+=i;
            }
        }
    }

    cout << "SOMA DOS IMPARES = " << soma;

    return 0;

}

File: Python/Lista_03(Estrutura_Repetitiva)/Soma_Impares.py
print("Digite dois numeros:")
x = int(input())
y = int(input())

soma = 0

if x>y:
    for i in range(y+1,x):
        if i%2!=0:
            soma += i
else:
    for i in range(x+1,y):
        if i%2!=0:
            soma += i

print(f"SOMA DOS IMPARES = {soma}")